include/bouncer.h

```diff
@@ -287,6 +289,13 @@ struct PgPool {
 	 */
 	struct StatList waiting_client_list;
 
+	/*
+	 * Clients that are waiting for a physical/logical replication connection to be
+	 * available. These requests are waiting for a new server connection to be
+	 * opened, before the request can be forwarded.
+	 */
+	struct StatList waiting_replication_client_list;
```

Comment:
Why is there a separate list for this? Why aren't these just part of the normal waiting_client_list? Maybe there is a reason, but it could be explained here. I mean, there is no separate list of active replication clients, so why a separate list of waiting replication clients?

Reply:
Good question. I added the reason to the comment now (also to the one for cancel connections since it's the same reason).

Reply:
I actually removed the state completely now. It turned out not to be necessary (given some other changes).